test/e2e/opaque_resource.go

```diff
@@ -38,7 +38,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", func() {
+var _ = framework.KubeDescribe("Opaque resources [Serial]", func() {
```

Comment:
The Serial was to allow for node cleanup without potential conflicts.  It's the safest thing you can do given that it was failing e2es.